lib/freq_estimator_impl.cc

```diff
@@ -57,6 +60,8 @@ namespace gr {
       const float *in = (const float *) input_items[0];
       float *out = (float *) output_items[0];
       float new_sig_avg;
+      const auto samp0_count = nitems_read(0);
```

Comment:
would be better if this is defined closer to where it is used first.

Reply:
actually that code belong to another MR (sorry, I failed to branch / did not double check) .. fixed now